timed/reports/views.py

```diff
@@ -182,6 +220,11 @@ def get_queryset(self):
             "verified_by"
         )
 
+        if CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
```

Comment:
Deny access by default, only allow if explicitly needed. This code only forbids to customers, but everyone else sees all the data